CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+1.2.1 / 2017-07-13
```

Comment:
You need an extra black line ahead of this or the markdown won't render correctly.